index.html

```diff
@@ -9,6 +9,17 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
+    <div class="mobile-menu">
+      <div class="mobile-menu-close">
+        <i class="mobile-menu-close-btn fas fa-times"></i>
+      </div>
+      <ul class="mobile-menu-options">
+        <li><a href="projects">Portoflio</a></li>
+        <li><a href="#about-me">About</a></li>
+        <li><a href="#contact">Contact</a>
+        </li>
```

Comment:
Kindly let's update the `mobile menu` UI so that it sticks to the design as much as possible. As it's the project requirement. You almost do a great job let's update a few more here: 

       Your Design                                Figma Design    

<img width="260" height="380" alt="image" src="https://user-images.githubusercontent.com/34578631/157398844-9bb84bc7-fa52-430d-8c5a-47b145dff79e.png">     <img width="260" height="380" alt="image" src="https://user-images.githubusercontent.com/34578631/157398967-d2b9489c-ad80-4058-982f-64ddcdf69540.png">

- Let's add a line break between each option as of the design
- Please change the close button ( `X` ) color